tests/test_app.py

```diff
@@ -445,13 +445,16 @@ def test_simple_index_list_name_with_underscore_no_egg(root, testapp):
 def test_json_info(root, testapp):
     root.join("foobar-1.0.zip").write("")
     root.join("foobar-1.1.zip").write("")
+    root.join("foobar-1.1-linux.zip").write("")
     root.join("foobarX-1.1.zip").write("")
 
     resp = testapp.get("/foobar/json")
     assert "info" in resp.json
     assert "releases" in resp.json
     assert len(resp.json["info"]) == 1
     assert len(resp.json["releases"]) == 2
+    assert len(resp.json["releases"]["1.0"]) == 1
+    assert len(resp.json["releases"]["1.1"]) == 2  # linux whl is included
```

Comment:
This is of course a styling-change (aka [[nit]](https://google.github.io/eng-practices/review/reviewer/standard.html#:~:text=%E2%80%9CNit%3A%20%E2%80%9C%20to%20let%20the%20author%20know%20that%20it%E2%80%99s%20just%20a%20point%20of%20polish%20that%20they%20could%20choose%20to%20ignore.)), but I'd suggest leaving the comment out, it's looking good in my opinion 😊 I think the test setup already communicates what's happening 👌 

```suggestion
    assert len(resp.json["releases"]["1.1"]) == 2 
```